.github/workflows/build_and_release.yml

```diff
@@ -136,7 +136,7 @@ jobs:
           make install
           mkdir -p /tmp/libtorrent-rasterbar
           [ -f /tmp/libtorrent-rasterbar/.unpack_ok ] || \
-            wget -qO- https://github.com/arvidn/libtorrent/archive/RC_1_2.tar.gz | \
+            wget -qO- https://github.com/arvidn/libtorrent/releases/download/libtorrent-1.2.10/libtorrent-rasterbar-1.2.10.tar.gz | \
```

Comment:
Why did you change RC_1_2 to 1.2.10?
All releases(Windows/MacOS/AppImage/whatever) is using RC_1_2 latest commit.

Reply:
> Why did you change RC_1_2 to 1.2.10?
> All releases(Windows/MacOS/AppImage/whatever) is using RC_1_2 latest commit.

Are you seriously?
I can't see any good reason to link qbittorrent against a **release candidate** version of its dependency instead of a stable release version.

Reply:
Yes, seriously.
Official also uses rc version for stable release.
https://www.qbittorrent.org/download.php
libtorrent-rasterbar | 1.2.6+git[a9968916ca](https://github.com/arvidn/libtorrent/commit/a9968916ca82366f1c236af59aaecb9bc94ffe73)
https://github.com/arvidn/libtorrent/releases/tag/libtorrent-1_2_6
commit: `b9b5443`

Well, IMO stable release don't mean to 100% stability for qbittorrent.
Just like #127 bug lol

Reply:
> Yes, seriously.
> Official also uses rc version for stable release.
> [qbittorrent.org/download.php](https://www.qbittorrent.org/download.php)
> libtorrent-rasterbar | 1.2.6+git[a9968916ca](https://github.com/arvidn/libtorrent/commit/a9968916ca82366f1c236af59aaecb9bc94ffe73)
> [arvidn/libtorrent@`libtorrent-1_2_6` (release)](https://github.com/arvidn/libtorrent/releases/tag/libtorrent-1_2_6)
> commit: `b9b5443`
> 
> Well, IMO stable release don't mean to 100% stability for qbittorrent.
> Just like #127 bug lol

fair enough.